ci/build_cpp.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
-# Copyright (c) 2020-2024, NVIDIA CORPORATION.
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
 
 set -euo pipefail
 
 rapids-configure-conda-channels
 
+# shellcheck source=/dev/null
```

Comment:
What does this mean? Is this something that ignores scripts that `shellcheck` cannot find?

Can we skip this check entirely, rather than add this comment in so many places?

Reply:
Yeah, this can definitely be ignored at a higher level to avoid the repetition